packages/flutter_tools/lib/src/application_package.dart

```diff
@@ -94,7 +95,7 @@ class AndroidApk extends ApplicationPackage {
   factory AndroidApk.fromApk(File apk) {
     final String aaptPath = androidSdk?.latestVersion?.aaptPath;
```

Comment:
I think we could improve this by first checking if `androidSdk == null` and printing a message that they don't even have the SDK.

Then we could check `if (!androidSdk.licensesAvailable) {` and print out a message saying they need to accept licenses.

Then we could check for the aaptPath, and print out a message saying aapt is missing and how to get it.

Reply:
And what would that message say exactly?